kyuubi-server/src/main/scala/org/apache/kyuubi/parser/plan/command/RunnableCommand.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.parser.plan.command
```

Comment:
what are parser/plan/command trying to grouping

Reply:
Kyuubi parser module  -> plan/node related class -> runnable command related class
<img width="407" alt="image" src="https://user-images.githubusercontent.com/51110188/206643517-6106b331-1dcd-4e3f-a9c3-84896364589b.png">

Reply:
and o.a.k.sql stands for?